ehrapy/anndata_ext.py

```diff
@@ -192,6 +196,177 @@ def get_column_values(adata: AnnData, indices: int | list[int]) -> np.ndarray:
     return np.take(adata.X, indices, axis=1)
 
 
+def type_overview(
+    data: MuData | AnnData, sort_by: str | None = None, sort_reversed: bool = False
+) -> None:  # pragma: no cover
+    """Prints the current state of an :class:`~anndata.AnnData` or :class:`~mudata.MuData` object in a tree format.
+    Output could be printed in sorted format by using one of `dtype`, `order`, `num_cats` or `None`, which sorts by data type, lexicographical order,
+    number of unique values (excluding NaN's) and unsorted respectively. Note that sorting by `num_cats` only affects
+    encoded variables currently and will display unencoded vars unsorted.
+
+    Args:
+        data: :class:`~anndata.AnnData` or :class:`~mudata.MuData` object to display
+        sort_by: How the tree output should be sorted. One of `dtype`, `order`, `num_cats` or `None` (defaults to None -> unsorted)
```

Comment:
```suggestion
        sort_by: How the tree output should be sorted. One of `dtype`, `order`, `num_cats` or None (defaults to None -> unsorted)
```

Can be missleading to use the quotes here for the None?

Reply:
good catch, yes